pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -36,6 +36,7 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [gpus_are_scarce](#gpus-are-scarce)                                   | Indicates whether to treat GPUs as a scarce resource in the cluster. |
 | [ip_detect_public_filename](#ip-detect-public-filename)               | The IP detect file to use in your cluster.  |
 | [master_discovery](#master-discovery)                                 | (Required) The Mesos master discovery method.         |
+| [master_external_loadbalancer](#master-external-loadbalancer)         | The DNS name for the load balancer.         |
```

Comment:
As @jgehrcke spotted the `master_external_loadbalancer` value can be also an IP address.

https://github.com/mesosphere/dcos-enterprise/blob/1.11/packages/bootstrap/extra/dcos_internal_utils/bootstrap.py#L2053-L2057

Reply:
fixed